tealium/collectors/momentsapi/MomentsAPIExtensions.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  MomentsAPIExtensions.swift
+//  tealium-swift
+//
+//  Copyright © 2024 Tealium, Inc. All rights reserved.
+//
+
+import Foundation
+#if momentsapi
+import TealiumCore
+#endif
+
+extension TealiumConfigKey {
+    static let momentsAPIRegion = "moments_api_region"
+    static let momentsAPIReferer = "moments_api_referer"
+}
+
+public enum MomentsAPIRegion: String {
+    // swiftlint:disable identifier_name
+    case germany = "eu-central-1"
+    case us_east = "us-east-1"
+    case sydney = "ap-southeast-2"
+    case oregon = "us-west-2"
+    case tokyo = "ap-northeast-1"
+    case hong_kong = "ap-east-1"
```

Comment:
Don't remember if we excluded the possibility to add support for potential new regions. 

In case an eventual last case with a custom string argument would prevent us from needing to update would solve the issue.